### ISSUE TITLE
CLI Execution

### ISSUE THREAD

**Issue by four43** [gjoseph92/geogif#2]
This is super slick! I love how quick and easy it is. Any interest in an install-able little executable we could just run from the CLI?

`geogif ./my_data.nc:TMP_2M ./out.gif`

Something like that?

**Comment by gjoseph92** [comment]
Yeah, that seems pretty handy!

I want to keep this library small and lightweight, so I wouldn't want to add anything for reading/opening various file formats. I'd probably say we'd support anything [`xarray.open_dataset`](https://xarray.pydata.org/en/stable/generated/xarray.open_dataset.html#xarray.open_dataset) (maybe also `open_rasterio` and `open_zarr`?) can handle. If people would need more complexity, I'd say it should be a separate library. The CLI part shouldn't become larger than the GIF part.

I won't have much time to work on this, but if you're interested I'd be happy to talk more if you want to add it.

**Comment by four43** [comment]
Yeah I think a simple little wrapper to grab args and just fire them into your main `gif()` method or w/e would be easy enough. I'll see if I get some time to fire over a PR.

Thanks!
